Source/AFError.swift

```diff
@@ -600,6 +700,8 @@ extension AFError.ServerTrustFailureReason {
             return "Certificate pinning failed for host \(host)."
         case let .publicKeyPinningFailed(host, _, _, _):
             return "Public key pinning failed for host \(host)."
+        case let .customEvaluationFailed(error):
+            return "Custom evaluation failed with error: \(error.localizedDescription)"
```

Comment:
A bit more detail would be good here: "Custom trust evaluation failed with error: "

Reply:
Done.